cinch/roles/jenkins_slave_container_wrapup/templates/jswarm.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -xe
-. /etc/sysconfig/jenkins_swarm.templated
+. /var/lib/jenkins/jenkins_swarm.templated
```

Comment:
You raise a good point about changing {{ jenkins_user_home }}.  We should strive to survive that scenario at least.  As for the path where we put this file, let's just use our best judgement and follow the [FHS](http://www.pathname.com/fhs/pub/fhs-2.3.html).